### PR TITLE
add(jigasi): ability to disable sip

### PR DESCRIPTION
**Pull request by debendraoli**

### CODE REVIEW

**Review by damencho — APPROVED** [review]

File: jigasi/rootfs/defaults/sip-communicator.properties [hunk 1]
```diff
@@ -120,6 +120,7 @@ net.java.sip.communicator.impl.protocol.jabber.acc{{ $index }}.BREWERY={{ $JIGAS
 net.java.sip.communicator.impl.protocol.jabber.acc{{ $index }}.DOMAIN_BASE={{ $XMPP_DOMAIN }}
 {{ end -}}
 org.jitsi.jigasi.BREWERY_ENABLED=true
+org.jitsi.jigasi.ENABLE_SIP={{ not .Env.JIGASI_DISABLE_SIP | default "true" }}
```
Comment (saghul) [inline jigasi/rootfs/defaults/sip-communicator.properties:123]: use `toBool` just in case someone sets it to 0 or 1.

**Comment by saghul** [comment]
Not sure about the precedence, do you need parentheses? `not (pipeline...)` ?

**Comment by debendraoli** [comment]
yeah that is little tricky, I guess It would be more readable if it is `ENABLE_SIP` ?

File: jigasi/rootfs/defaults/sip-communicator.properties [hunk 2]
```diff
@@ -8,6 +8,7 @@
 {{ $XMPP_PORT := .Env.XMPP_PORT | default "5222" -}}
 {{ $XMPP_SERVER := .Env.XMPP_SERVER | default "xmpp.meet.jitsi" -}}
 {{ $XMPP_SERVERS := splitList "," $XMPP_SERVER -}}
+{{ $DISABLE_SIP := .Env.JIGASI_DISABLE_SIP | default "true" | toBool -}}
```
Comment (saghul) [inline jigasi/rootfs/defaults/sip-communicator.properties:11]: this needs to default to false now

Comment (debendraoli) [reply jigasi/rootfs/defaults/sip-communicator.properties:11]: yeah. missed that

**Review by saghul — APPROVED** [review]

**Pull request merged**